src/main/java/de/hpi/bpt/argos/core/Application.java

```diff
@@ -30,8 +30,8 @@ public static void main(String[] args) {
 
 			PropertyEditorImpl.getInstance().setProperty(splitArg[0], splitArg[1]);
 		}
-		(new ArgosImpl()).start();
 
+		Argos argos = ArgosImpl.run();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "argos" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "argos". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1854)